tests/test_command.py

```diff
@@ -34,9 +34,9 @@ def mock_error(self, message):
     raise ValueError(message)
 
 
-argparse.ArgumentParser.error = mock_error
+argparse.ArgumentParser.error = mock_error  # type: ignore[assignment]  # Cannot assign to a method
```

Comment:
argparse encourages inheriting from ArgumentParser, instead of doing this. That would be however quite inconvenient to do (although python's unittest.mock should be able to get me what I need). I decided to suppress, just to be done with it. It was wrong since forever, so it might just as well stay "wrong" (in mypy's estimation).